src/hooks/use-sync-states-progress-info.ts

```diff
@@ -30,12 +35,42 @@ export function useSyncStatesProgressInfo() {
 			finished: {
 				key: 'finished',
 				progress: 100,
-				message: __( 'Backup imported' ),
+				message: __( 'Pull complete' ),
+			},
+			failed: {
+				key: 'failed',
+				progress: 100,
+				message: __( 'Error pulling changes' ),
+			},
+		} as const;
+	}, [ __ ] );
+
+	const pushStatesProgressInfo = useMemo( () => {
+		return {
+			creatingBackup: {
+				key: 'creatingBackup',
+				progress: 30,
+				message: __( 'Creating backup…' ),
+			},
+			uploading: {
+				key: 'uploading',
+				progress: 50,
+				message: __( 'Pushing changes…' ),
```

Comment:
Would it make sense to change this one to show a more accurate state?
```suggestion
				message: __( 'Uploading Studio site…' ),
```

Reply:
done https://github.com/Automattic/studio/pull/662/commits/fb4700e30e97661011d04bcc662fc0ab4a6a8ede